tools/link.py

```diff
@@ -1659,6 +1659,8 @@ def phase_linker_setup(options, state, newargs):
     # For chrome see: https://crbug.com/324992397
     if settings.MIN_CHROME_VERSION != feature_matrix.UNSUPPORTED and settings.MEMORY64 and settings.MAXIMUM_MEMORY > 2 ** 32:
       settings.WEBGL_USE_GARBAGE_FREE_APIS = 0
+    if settings.WEBGL_USE_GARBAGE_FREE_APIS and settings.MIN_WEBGL_VERSION >= 2:
```

Comment:
Can this check be simplified, given that `WEBGL_USE_GARBAGE_FREE_APIS` is only set when `MIN_WEBGL_VERSION >= 2` anyhow?

Reply:
```suggestion
    if settings.WEBGL_USE_GARBAGE_FREE_APIS:
      assert(settings.MIN_WEBGL_VERSION >= 2)
```

Reply:
Are you confusing `MAX_WEBGL_VERSION` (which is the setting which `WEBGL_USE_GARBAGE_FREE_APIS` depends on) with `MIN_WEBGL_VERSION` which is used here?

Reply:
Ah, yes, that was exactly my confusion, thanks!